src/components/rating/_rating.scss

```diff
@@ -62,8 +61,8 @@ $includeHtml: false !default;
     }
 
     &--small {
-      @include fixText($rateSmallFontSize);
```

Comment:
Its already added in line 74.

It was centring element in chrome but in a way that if button was 20px and rate-box was 24px it was sticking out 2px from top and 2px from bottom. On Safari it was sticking 4px from bottom - thats why button were broken.